Build Error E0275

```
   Compiling weechat-matrix v0.1.0 (/src/weechat-matrix-rs)                                                                                                    
error[E0275]: overflow evaluating the requirement `(std::string::String, matrix_sdk::encryption::vodozemac::olm::DecryptionError): std::marker::Send`                                 
    --> src/connection.rs:128:17                                                                                                                                                      
     |                                                                                                                                                                                
 128 |         runtime.spawn(Connection::sync_loop(                                                                                                                                   
     |                 ^^^^^                                                                                                                                                          
     |                                                                                                                                                                                
     = help: consider increasing the recursion limit by adding a `#![recursion_limit = "256"]` attribute to your crate (`matrix`)                   
```

--- a/src/lib.rs
+++ b/src/lib.rs
@@ -1,3 +1,6 @@
+// Increase recursion limit to pre-rustc 1.94 level for macro expansion and auto-dereferencing.
+#![recursion_limit = "256"]
+
 mod bar_items;
 mod commands;
 mod completions;

